### PR TITLE
Issue #3564 fixed

### PR DESCRIPTION
**Pull request by vishal-webkul**
**BUGS:**

issue: #3564 

**For things other than bugs:**

> Describe that thing in very short line, word limit is 200.
> Otherwise use **issue #id** if the issue was filed as **feature** request.

### CODE REVIEW

**Comment by vaishaliwebkul** [comment]
@vishal-webkul 
image swatch is not visible images while editing attribute

**Comment by vishal-webkul** [comment]
@vaishaliwebkul 

edit problem resolved

File: packages/Webkul/Admin/src/Resources/views/catalog/attributes/edit.blade.php [hunk 1]
```diff
@@ -371,10 +371,11 @@
...
-                                    <input type="text" v-validate="'required'" v-model="row['admin_name']" :name="adminName(row)" class="control" data-vv-as="&quot;{{ __('admin::app.catalog.attributes.admin_name') }}&quot;"/>
-                                    <span class="control-error" v-if="errors.has(adminName(row))">@{{ errors.first(adminName(row)) }}</span>
+                            <td style="white-space: nowrap;" v-if="show_swatch && swatch_type == 'image'">
+                                <div class="control-group" :class="[errors.has('options[' + row.id + '][swatch_value]') ? 'has-error' : '']">
+                                    <img style="width: 36px;height: 36px;vertical-align: middle;background: #F2F2F2;border-radius: 2px;margin-right: 10px;" v-if="row.swatch_value_url" :src="row.swatch_value_url"/>
+                                    <input type="file" v-validate="'size:600'" accept="image/*" :name="'options[' + row.id + '][swatch_value]'"/>
+                                    <span class="control-error" v-if="errors.has('options[' + row.id + '][swatch_value]')">The image size must be less than 600 KB</span>
```
Comment (rahulcs0082) [inline packages/Webkul/Admin/src/Resources/views/catalog/attributes/edit.blade.php:371]: Kindly use translations.

File: packages/Webkul/Admin/src/Resources/views/catalog/attributes/create.blade.php [hunk 2]
```diff
@@ -296,7 +296,10 @@
                             </td>
 
                             <td v-if="show_swatch && swatch_type == 'image'">
-                                <input type="file" accept="image/*" :name="'options[' + row.id + '][swatch_value]'"/>
+                                <div class="control-group" :class="[errors.has('options[' + row.id + '][swatch_value]') ? 'has-error' : '']">
+                                    <input type="file" v-validate="'size:600'" accept="image/*" :name="'options[' + row.id + '][swatch_value]'"/>
+                                    <span class="control-error" v-if="errors.has('options[' + row.id + '][swatch_value]')">The image size must be less than 600 KB</span>
```
Comment (rahulcs0082) [inline packages/Webkul/Admin/src/Resources/views/catalog/attributes/create.blade.php:301]: translation is missing.

**Comment by rahulcs0082** [comment]
Hi @vishal-webkul 

Use the below code to avoid translation issue.

`<td v-if="show_swatch && swatch_type == 'image'">
                                <div class="control-group" :class="[errors.has('options[' + row.id + '][swatch_value]') ? 'has-error' : '']">
                                    <input type="file" v-validate="'size:600'" accept="image/*" :name="'options[' + row.id + '][swatch_value]'" data-vv-as="&quot;{{ __('admin::app.catalog.attributes.image-swatch') }}&quot;"/>
                                    <span class="control-error" v-if="errors.has('options[' + row.id + '][swatch_value]')">@{{ errors.first('options[' + row.id + '][swatch_value]') }}</span>
                                </div>
                            </td>`

**Pull request merged**